actionpack/test/dispatch/cookies_test.rb

```diff
@@ -603,7 +603,7 @@ def test_encrypted_cookie_using_hybrid_serializer_can_migrate_marshal_dumped_val
     secret = key_generator.generate_key(encrypted_cookie_salt)
     sign_secret = key_generator.generate_key(encrypted_signed_cookie_salt)
 
-    marshal_value = ActiveSupport::MessageEncryptor.new(secret, sign_secret, serializer: Marshal).encrypt_and_sign("bar")
+    marshal_value = ActiveSupport::MessageEncryptor.new(secret[0, ActiveSupport::MessageEncryptor.key_len], sign_secret, serializer: Marshal).encrypt_and_sign("bar")
```

Comment:
@vipulnsward `ActiveSupport::MessageEncryptor.key_len` is repeated on multiple tests, can we define a method say `max_key_length` or something similar? Or is it an overkill to DRY this up in tests?